src/qt/guiutil.cpp

```diff
@@ -1659,6 +1659,19 @@ QString NetworkToQString(Network net)
     assert(false);
 }
 
+QString ConnectionTypeToQString(ConnectionType conn_type)
```

Comment:
I'd suggest to backport bitcoin-core/gui#180 also

But that's not a blocker to get this one merged, because there's no critical fixes, just documentation updates.